examples/agentworkflow/blog_writer.ts

```diff
@@ -0,0 +1,85 @@
+import { OpenAI } from "@llamaindex/openai";
+import {
+  AgentToolCall,
+  AgentToolCallResult,
+  AgentWorkflow,
+  FunctionAgent,
+} from "@llamaindex/workflow";
```

Comment:
@thucpn @leehuwuj what do you think about that?
```suggestion
} from "@llamaindex/workflow/agent";
```